<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_Time">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(11, 15, 255);</string>
    </property>
    <property name="text">
     <string>date(now)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_DecknameMine">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(11, 15, 255);</string>
    </property>
    <property name="text">
     <string>my deck name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_DecknameEnemy">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(11, 15, 255);</string>
    </property>
    <property name="text">
     <string>enemy deck name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Attackerside">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(11, 15, 255);</string>
    </property>
    <property name="text">
     <string>attacker side</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_WinLose">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(11, 15, 255);</string>
    </property>
    <property name="text">
     <string>WIN / LOSE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Reason">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>310</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(11, 15, 255);</string>
    </property>
    <property name="text">
     <string>REASON</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_First">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>first</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Second">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>420</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>second</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_DecknameMine">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_DecknameEnemy">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>180</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Button</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameInput">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Input Deck Name</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Win">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>380</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>WIN</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Lose">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>420</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LOSE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Reason">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>390</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Input Win/Lose reason</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
